src/test/K6/src/tests/platform/authorization/delegations/delegations.js

```diff
@@ -52,13 +52,13 @@ export default function (data) {
     coveredBy: 'urn:altinn:userid',
     resource: ['urn:altinn:app', 'urn:altinn:org', 'urn:altinn:task'],
   };
-  res = delegation.addRules(altinnToken, policyMatchKeys, 111, 123, 456, appOwner, appName, 'Task_1', 'read');
+  res = delegation.addRules(altinnToken, policyMatchKeys, 20001340, 50001340, 500001341, appOwner, appName, 'Task_1', 'confirm');
```

Comment:
seems like the same partyIds are repeated here. would it be better to make these configurable for the tests to be able to run smooth in all environments?